pkg/webhook/webhook.go

```diff
@@ -319,10 +371,14 @@ func validateDeployment(appInfo v1alpha1.ApplicationParams, kubeClient kubernete
 		LabelSelector: appInfo.Applabel,
 	})
 	if err != nil {
-		return false, fmt.Errorf("unable to list deployments, please provide a suitable RBAC with apiGroup 'apps', and resource 'deployments' and verb 'list' , or remove this deployment")
+		return false, fmt.Errorf("unable to list deployments, please provide a suitable RBAC with apiGroup 'apps', resource 'deployments' and verb 'list' , or remove this deployment")
```

Comment:
This is one case. It is also possible the RBAC is fine & the right labels are not provided/